projects/components/src/utils/test/widget-object/angular/angular-widget-object-element.ts

```diff
@@ -95,48 +114,60 @@ export class AngularLocatorDriver implements LocatorDriver<TestElement> {
 export class TestElement implements Iterable<TestElement> {
     constructor(public elements: DebugElement[], public fixture: ComponentFixture<any>) {}
 
+    private get firstNativeElement(): HTMLElement {
+        return this.firstDebugElement.nativeElement;
```

Comment:
Should this or firstDebugElement throw an error if there's more than one element present? Might catch an error where there's a query that returns multiple elements but the tester thought there should only be one